src/cc65/opcodes.h

```diff
@@ -114,6 +114,7 @@ typedef enum {
     OP65_SED,
     OP65_SEI,
     OP65_STA,
+    OP65_STP, /* 65c02, 65816 stop */
```

Comment:
Please move the comment over to character column 33 (that is, align it with the other comments).

Reply:
Now jump on one leg ;-)